app/models/integrations/anrok_integration.rb

```diff
@@ -2,6 +2,10 @@
 
 module Integrations
   class AnrokIntegration < BaseIntegration
+    has_many :error_details, -> { where({error_details: {error_code: 'tax_error'}}) },
```

Comment:
I think we don't need it in this direction since it would return all error details included in all invoices, I think that having access on invoice to fetch error details is enough

Reply:
I thought we might need it when showing all errors on the integration page 🤔

Reply:
I think we will display only number of failed errors, but we can leave it for now